HACKING.md

```diff
@@ -47,7 +51,8 @@ to:
 --exec-opt native.cgroupdriver=cgroupfs \
 ```
 
-Then, restart the docker daemon:
+Restart the docker daemon for the override
+to take effect:
```

Comment:
Note you should be able to drop the `daemon-reload` step now, since `systemctl edit` already does this.

Reply:
Tried it without - works!